scanapi/evaluators/code_evaluator.py

```diff
@@ -19,6 +19,23 @@ class CodeEvaluator:
 
     @classmethod
     def evaluate(cls, sequence, spec_vars, is_a_test_case=False):
+        """Evaluate a sequence of python code and return a evaluation indicator.
+
+        Args:
+            sequence[string]: string representing python statements to be
+            evaluted
+            spec_vars[dict]: dictionary containing response for an API request
+            is_a_test_case[bool]: indicator for checking if the given evalution
+            is a test case
+
+        Returns:
+            Tuple[Boolean, string]: (True, None) for successful evalution
+
+        Raises:
+            InvalidPythonCodeError: If receives invalid python statements
```

Comment:
👏

Reply:
@sijanonly Your suggestion of having something like a returns list of multiple values looks a lot cleaner. Let's do that. 😄 

```
Returns:
        tuple: a tuple containing:
            -  [Boolean]: True for valid evalution
            -  [string]: None for valid evalution, otherwise the code itself
```

Reply:
> @sijanonly Your suggestion of having something like a returns list of multiple values looks a lot cleaner. Let's do that. smile
> 
> what do you think @camilamaia ?
> 
> ```
> Returns:
>         tuple: a tuple containing:
>             -  [Boolean]: True for valid evalution
>             -  [string]: None for valid evalution, otherwise the code itself
> ```

Okay great  :) . I will do that.